up-core-api/release/conanfile.py

```diff
@@ -0,0 +1,71 @@
+from conan import ConanFile
+from conan.tools.cmake import CMakeToolchain, CMake, cmake_layout, CMakeDeps
+from conan.tools.scm import Git
+from conan.tools.files import copy, get
+import os
+import yaml
+
+class upCoreApiRecipe(ConanFile):
+    name = "up-core-api"
+
+    # Optional metadata
+    license = "<Put the package license here>"
+    author = "<Put your name here> <And your email here>"
+    url = "<Package recipe repository url here, for issues about the package>"
+    description = "<Description of hello package here>"
+    topics = ("<Put some tag here>", "<here>", "<and here>")
```

Comment:
What do you think about using this for the metadata?

```suggestion
    license = "Apache-2.0"
    author = "Contributors to the Eclipse Foundation <uprotocol-dev@eclipse.org>"
    url = "https://github.com/eclipse-uprotocol/up-spec"
    description = "Provides the uProtocol data model and core services definitions compiled for C++ from source proto files"
    topics = ("automotive", "iot", "uprotocol", "messaging")
```

Reply:
LGTM